api/app/services/pending_response_service.py

```
@@ -0,0 +1,360 @@
+                        continue
+
+            if not updated:
+                return False
+
+            # Write back all entries
+            with open(self.pending_file, "w") as f:
+                for entry in entries:
+                    f.write(json.dumps(entry) + "\n")
+
+            logger.info(f"Updated response {response_id} to status={status}")
+            return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)